bouncer/sentry.py

```diff
@@ -0,0 +1,53 @@
+"""Report exceptions to Sentry/Raven."""
+from pyramid import tweens
+import raven
+
+from bouncer import __about__
+
+
+#: The Raven client object that's used to send reports to Sentry.
+#: This object is created once per app and re-used across requests.
+#:
+#: This should be accessed via request.raven rather than directly, because
+#: get_raven_client() below adds per-request context to the client.
+raven_client = raven.Client(release=__about__.__version__)
```

Comment:
I've gone ahead and [done this on master](https://github.com/hypothesis/bouncer/commit/f2fb1b6b2f76445d303753f6ba6bc7cbbcafbcb4)